app/styles/identity-card.module.css

```diff
@@ -0,0 +1,35 @@
+
+.verification-page{
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+    align-items: center;
+}
+
+.verification-page__heading-Icon{
+    display: flex;
+}
+
+.hourglass-half-icon{
+    padding: 1.4rem 0 0 0.5rem;
+    color: #926829;

```

Comment:
Try using color variables here

Reply:
Done with changes...can you please check latest comment @satyam73